config/application.rb

```diff
@@ -5,7 +5,7 @@
 require "active_model/railtie"
 require "active_job/railtie"
 require "active_record/railtie"
-# require "active_storage/engine"
+require "active_storage/engine"
```

Comment:
I've mentioned this in a response to an outdated comment, but could you add something to the commit message about what failed when this was commented out?

Reply:
I had a think - and a Google - and we shouldn't have to enable `active_storage` just to satisfy some new defaults. Yet, when I ran the tests it failed with an error indicating exactly that. I've now re-built the app `make local-links-manager` and now it seems fine - all tests pass (with and without docker), app loads and runs correctly.

I can only assume it was something 'sticky' in the docker image.

Reply:
Thanks for testing the app loading. It's always a bit of a hassle with the admin apps, but good to do.